client/remote.go

```diff
@@ -137,6 +137,11 @@ func healthchecker(c *Conn) {
 	}
 }
 
+// GetConnPool allows other types of remotes to interact with the connection pool
+func GetConnPool() *connPoolType {
```

Comment:
i agree with the lint here, but i suppose practically speaking this return value is just being passed back into this package so it's probably not an actual issue

Reply:
Yea, not an issue but it's a reasonable lint